test/test_matmul_cuda.py

```diff
@@ -374,21 +374,24 @@ def _test_tautological_mm(self, device: str = "cuda",
     @unittest.skipIf(not scaled_mm_supported_device(), f8_msg)
     def test_float8_basics(self, device) -> None:
         self._test_tautological_mm(device, e4m3_type, e4m3_type, size=16)
-        # hipblaslt does not yet support mixed e4m3_type input
         if torch.version.hip is None:
-            self._test_tautological_mm(device, e4m3_type, e5m2_type, size=32)
-            self._test_tautological_mm(device, e5m2_type, e4m3_type, size=48)
-        # According to https://docs.nvidia.com/cuda/cublas/#id99 8F_E5M2 MM is unsupported
-        with self.assertRaises(RuntimeError):
+            # According to https://docs.nvidia.com/cuda/cublas/#id99 8F_E5M2 MM is unsupported
```

Comment:
why the CUDA case is change? how did this change come in? due to git merge?

Reply:
here is some rearrange for tests. 
some of tests are passed on rocm now: 
source: https://github.com/ROCm/pytorch/commit/9f765703749d5c7744433241057f0df9601b8798

Reply:
cherry-picked from https://github.com/ROCm/pytorch/commit/9521966dfed08c968507888a26f0c502a576d9a9